features/layout/blocks/banner/BannerBlock.js

```diff
@@ -48,6 +48,7 @@ const BannerBlock = ({ block: { banner } }) => {
 
   const linkProps = LINK[banner];
   const Link = linkProps.route ? LocalLink : ExternalLink;
+  const imageLoading = banner === NY2021 ? 'eager' : 'lazy';
```

Comment:
1. `ny2021` slug is lower-cased
2. Is it based on current index page layout? Let's not rely on it and pass `imageLoading` into component as you do with other blocks.

Reply:
`NY2021` - is constant